src/pages/LoginPage/LoginPage.page.jsx

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import { useForm } from "react-hook-form";
+import { Button } from "../../StyledComponents/Button";
+import { Input } from "../../StyledComponents/Input";
+import { ButtonsContainer, ErrorMsg, FormContainer, InputsContainer } from "./Login.styles";
+
+export default function Login() {
+  const {
+    login,
```

Comment:
`useForm()` contiene una función `register()`, no es un "register" de usuarios, es para indicar que el input que use ese `register()` forma parte de un formulario. Se puede decir que "registra" el campo a un formulario.


```suggestion
    register,
```